src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/test/responses/PrisonerSearchResponses.kt

```diff
@@ -120,7 +120,7 @@ fun prisonerSearchResponse(responseSetup: ApiResponseSetup) = """
     ],
     "addresses": [
       {
-        "fullAddress": "1 Main Street, Crookes, Sheffield, South Yorkshire, S10 1BP, England",
+        "fullAddress": "${responseSetup.addresses[0].fullAddress}",
```

Comment:
Small point, but we could now create a dynamic list of 'addresses' for this so we can test multiple addresses being stored. Rather than just using the first one in the list.